### DISCUSSION TITLE
[Feature Request] Pinned Notes

### DISCUSSION THREAD

**Discussion by ghost** [GitJournal/GitJournal#945] [Feature Requests]
I like to have a "To Do" note at the very top of my notes list. Many other note-taking apps offer this, and I think it would really help the user experience for GitJournal. This way, when you have multiple notes, you can make it so that certain ones are always at the top of your screen, easy to find.

You could have a .yml file stored in the Git repo with an array of pinned notes based off of the file name. You'd have to make sure that whenever you update the note's file name in the app, you update the .yml file though.

Any thoughts on this as a feature?

**Comment by vHanda** [comment]
Edit the note metadata in the raw editor and add pinned: true

I haven't yet exposed it in the UI

**Reply by ghost** [reply]
Thanks for the solution, works great!

**Reply by ghost** [reply]
What other metadata variables are available for people to use?

**Reply by vHanda** [reply]
That's the only one that isn't exposed via a UI.
